src/shared/components/deletion-confirmation/DeletionConfirmation.scss

```diff
@@ -0,0 +1,18 @@
+@import 'src/styles/common';
+
+.deleteConfirmationContainer {
+  display: flex;
+  align-items: center;
+  justify-content: var(--container-justify-content, flex-end);
+  column-gap: var(--container-column-gap, 30px);
+  margin: var(--container-margin, 20px 0px);
+}
+
+.warningText {
+  color: var(--warning-text-color, #{$red});
```

Comment:
`--deletion-confirmation-warning-color`

Reply:
but the css will be inside the component css and will have the component name attached to the class. I am not sure why you will need the component name with the css variable. the way we used our css variable is closely coupled with the react component so dont see where the confusion will be

Reply:
> but the css will be inside the component css and will have the component name attached to the class

If you use CSS variables to modify the behaviour of your component, it means that the component that will actually set this variable will be one of the parents of your component.

![image](https://user-images.githubusercontent.com/6834224/190425186-e5ea197f-ca64-44b5-aa86-0b45819966f3.png)

Reply:
Agreed from slack discussion that css variable should be unique so that we can identify them further up in the tree and also we can more than 1 variable doing the same thing (changing color) but in different component